### PR TITLE
Revert "type:docs making updates to the readme and script.js file to make the…"

### PR DESCRIPTION
**Pull request by sarah-udacity**
Reverts udacity/cd2073-intro-to-js-1-project-starter#1

### CODE REVIEW

**Comment by sarah-udacity** [comment]
This pull request needs changes to scripts.test.js and front.js before it can be included.

File: starter/src/assets/script.js [hunk 1]
```diff
@@ -14,60 +12,80 @@
...
+  
+
+/* Create an array named productsArr and add all product object literals to array */
+
+
+/* Declare an empty array named cartArr to hold the items in the cart */
+
 
-/* Declare an empty array named cart to hold the items in the cart */
+// Helper function to get product
```
Comment (sarah-udacity) [inline starter/src/assets/script.js:23]: I'm not sure what you mean here.  Please clarify.

File: README.md [hunk 2]
```diff
@@ -107,10 +106,10 @@ Step 1: Review the Starter Code
 Step 2: Build the Products
 Step 3: Create the Cart Functionality
 
-- `addProductToCart()`
-- `increaseQuantity()`
-- `decreaseQuantity()`
-- `removeProductFromCart()`
+- `addToCart()`
+- `increase()`
```
Comment (sarah-udacity) [inline README.md:110]: front.js and script.test.js will need to be updated with these new function names

File: starter/src/assets/script.js [hunk 3]
```diff
@@ -1,11 +1,9 @@
-/* Create an array named products which you will use to add all of your product object literals that you create in the next step. */
-
 /* Create 3 or more product objects using object literal notation 
    Each product should include five properties
    - name: name of product (string)
    - price: price of product (number)
-   - quantity: quantity in cart should start at zero (number)
-   - productId: unique id for the product (number)
```
Comment (sarah-udacity) [inline starter/src/assets/script.js:8]: productId works -- but SKU is a real-world term for the product Id. I'm not convinced this is a necessary or helpful change.

File: starter/src/assets/script.js [hunk 4]
```diff
@@ -14,60 +12,80 @@
...
 
-/* Create a function named addProductToCart that takes in the product productId as an argument
-  - addProductToCart should get the correct product based on the productId
-  - addProductToCart should then increase the product's quantity
+
+// Gets the product from the SKU received from the frontend
+
+
+/* Create a function named addToCart that takes in the product SKU as an argument
+  - addToCart should get the correct product based on the SKU
```
Comment (sarah-udacity) [inline starter/src/assets/script.js:31]: Again, not sure that productId is a helpful change.

File: starter/src/assets/script.js [hunk 5]
```diff
@@ -14,60 +12,80 @@
...
+  - addToCart should then increase the product's quantity
   - if the product is not already in the cart, add it to the cart
 */
 
-/* Create a function named increaseQuantity that takes in the productId as an argument
-  - increaseQuantity should get the correct product based on the productId
-  - increaseQuantity should then increase the product's quantity
+
+/* Create a function named increase that takes in the product SKU as an argument
+  - increase should get the correct product based on the SKU
```
Comment (sarah-udacity) [inline starter/src/assets/script.js:38]: Similar issue with productId instead of SKU

File: starter/src/assets/script.js [hunk 6]
```diff
@@ -14,60 +12,80 @@
...
+  - increase should then increase the product's quantity
 */
 
-/* Create a function named decreaseQuantity that takes in the productId as an argument
-  - decreaseQuantity should get the correct product based on the productId
-  - decreaseQuantity should decrease the quantity of the product
+
+/* Create a function named decrease that takes in the product SKU as an argument
+  - decrease should get the correct product based on the SKU
```
Comment (sarah-udacity) [inline starter/src/assets/script.js:44]: Similar issue with productId instead of SKU

File: starter/src/assets/script.js [hunk 7]
```diff
@@ -14,60 +12,80 @@
...
   - if the function decreases the quantity to 0, the product is removed from the cart
 */
 
-/* Create a function named removeProductFromCart that takes in the productId as an argument
-  - removeProductFromCart should get the correct product based on the productId
-  - removeProductFromCart should update the product quantity to 0
-  - removeProductFromCart should remove the product from the cart
+
+/* Create a function named remove that takes in the product SKU as an argument
+  - remove should get the correct product based on the SKU
```
Comment (sarah-udacity) [inline starter/src/assets/script.js:51]: Similar issue with productId instead of SKU

File: starter/src/assets/script.js [hunk 8]
```diff
@@ -14,60 +12,80 @@
...
 */
 
-/* Create a function called emptyCart that empties the products from the cart */
 
 /* Create a function named pay that takes in an amount as an argument
   - pay will return a negative number if there is a remaining balance
   - pay will return a positive number if money should be returned to customer
 */
 
-/* Place stand out suggestions here (stand out suggestions can be found at the bottom of the project rubric.)*/
```
Comment (sarah-udacity) [inline starter/src/assets/script.js:55]: Not sure why this hint is removed.

File: starter/src/assets/script.js [hunk 9]
```diff
@@ -14,60 +12,80 @@
...
-   addProductToCart,
-   increaseQuantity,
-   decreaseQuantity,
-   removeProductFromCart,
+   productsArr,
+   cartArr,
+   addToCart,
+   increase,
+   decrease,
+   remove,
```
Comment (sarah-udacity) [inline starter/src/assets/script.js:88]: As noted above, the script.tests.js and front.js files will need to be updated with these new exports.

**Review by sarah-udacity — COMMENTED** [review]
See items that need to be addressed.

File: starter/src/assets/script.js (hunk 8 above)
Comment (jaroot32) [reply starter/src/assets/script.js:55]: For this one I was just hoping to add in parenthesis where the standout suggestions can be found, for example - (stand out suggestions can be found at the bottom of the project rubric.)

File: starter/src/assets/script.js (hunk 3 above)
Comment (jaroot32) [reply starter/src/assets/script.js:8]: Ok, but how about starting the quantity at 0 and changing integer to number?

File: starter/src/assets/script.js (hunk 1 above)
Comment (jaroot32) [reply starter/src/assets/script.js:23]: I was thinking there is no where in the curriculum where the concept of a helper function is explained and ultimately it is not necessary to write each function.  I think it is a place where some students could get blocked and discouraged very early in attempting the project.  Perhaps we could have the concept of refactoring parts of the core functions into a helper function in the stand out suggestions?

File: README.md (hunk 2 above)
Comment (jaroot32) [reply README.md:110]: I will work on this and make a new pull request.

File: starter/src/assets/script.js (hunk 4 above)
Comment (jaroot32) [reply starter/src/assets/script.js:31]: I agree, my team lead had requested this change but I will relay your comment back and let them know.  Thanks!

File: starter/src/assets/script.js (hunk 1 above)
Comment (sarah-udacity) [reply starter/src/assets/script.js:23]: That works

File: starter/src/assets/script.js (hunk 3 above)
Comment (sarah-udacity) [reply starter/src/assets/script.js:8]: Yep. That works

File: starter/src/assets/script.js (hunk 8 above)
Comment (sarah-udacity) [reply starter/src/assets/script.js:55]: Ok.  We can do that

**Pull request merged**